docs/topics/proc_releasing-launcher-docs.adoc

```diff
@@ -39,7 +39,37 @@ Replace `$REMOTE` with the name of the upstream remote.
 --
 
 . Request publication:
-.. File a pull request in the link:https://github.com/openshiftio/saas-launchpad/blob/master/launchpad-services/launcher-documentation.yaml#L2[openshiftio/saas-launchpad] repository, where you change the `hash` value to the hash of the commit you want to publish from the link:{link-repo-docs}[{name-repo-docs}] repository. Usually, this will be the latest commit in the `master` branch.
+.. Fork link:https://access.redhat.com/documentation/en-us/red_hat_jboss_web_server/5.0/html/red_hat_jboss_web_server_for_openshift/index[openshiftio/saas-launchpad].
```

Comment:
```suggestion
.. Fork link:https://github.com/openshiftio/saas-launchpad[openshiftio/saas-launchpad].
```
Te current URL does not seem to link to the `saas-launchpad` repo on GitHub. It seems to point to JWS 5.0 docs.

Reply:
Thanks!